### DISCUSSION TITLE
Made a custom theme, easier than I expected

### DISCUSSION THREAD

**Discussion by kt-alt** [Lissy93/dashy#1291] [Show and tell]
Thanks @Lissy93 and team.  

Just started with Dashy and like it alot.  Keep up the awesome work.  The great docs and well laid out theming system made it easy to create a custom one.  I packaged it up and published it to github in case it can help others here: 
[https://github.com/kt-alt/dashy-tama-theme](https://github.com/kt-alt/dashy-tama-theme)

The theme is based on the built-in Adventure theme and has the following additions:

- a random background image from [Unsplash.com](https://unsplash.com/) (on every page load) and can be customized
- added transparent background tile for Widgets (to match the Items one)
- transparent header & footer (and other minor stuff)

![dashy_tama_screenshot](https://raw.githubusercontent.com/kt-alt/dashy-tama-theme/fad12d926e8a5dcbce6371a32adab5788a22d569/screenshot/dashy_tama_screenshot.png)

Cheers

**Comment by whdoughty1976** [comment]
I just dl Dashy... How do I even get started? Is there a guide to help me? I know literally nothing about github.

**Comment by CrazyWolf13** [comment]
> I just dl Dashy... How do I even get started? Is there a guide to help me? I know literally nothing about github. 

Yes there is one and even one of the best documentations, lissy really put very much effor into them!

https://dashy.to/docs

**Comment by sushibait** [comment]
Really gorgeous theme. I'm getting here about 2 years late, but looks very nice!
